indico/modules/rb_new/client/js/modules/calendar/Calendar.jsx

```diff
@@ -39,6 +39,7 @@ import BookFromListModal from '../../components/modals/BookFromListModal';
 
 
 import '../../components/Timeline.module.scss';
+import TimelineHeader from '../../components/TimelineHeader';
```

Comment:
We normally keep the CSS import at the bottom. It's not enforced by ESLint, but it makes it easier to figure out what is CSS vs what is JS.